src/main.tsx

```diff
@@ -13,7 +13,7 @@ import { store } from './state/_store';
 
 const Router = import.meta.env.VITE_ROUTER_TYPE === 'hash' ? HashRouter : BrowserRouter;
 
-ReactDOM.createRoot(document.getElementById('root') as HTMLElement).render(
+ReactDOM.createRoot(document.getElementById('root')!).render(
```

Comment:
In this case I think it's fine because it really is basically impossible for that element to not exist and there's really nothing we can do if it's not there.